project/src/components/main-page/main-page.tsx

```diff
@@ -1,18 +1,25 @@
 import Header from '../header/header';
-import {Link} from 'react-router-dom';
 import {OfferType} from '../../types/offer';
 import PlaceCardList from '../place-card-list/place-card-list';
 import Map from '../map/map';
+import LocationList from '../location-list/location-list';
+//import { useAppSelector } from '../../hooks';
 import {useState} from 'react';
+//import {filterCity} from '../../utils';
 
 type MainPageProps = {
-  rentOffer: number;
   offers: OfferType[];
 }
 
-function MainPage({rentOffer, offers}:MainPageProps): JSX.Element {
+function MainPage({offers}:MainPageProps): JSX.Element {
```

Comment:
```suggestion
function MainPage(): JSX.Element {
```
тебе оферы нужно сразу уже со стора добавлять